### DISCUSSION TITLE
Rider - Automatically Nesting JSON Configuration Files | Dan Does Code

### DISCUSSION THREAD

**Discussion by giscus** [danielmackay/nextjs-personal-blog#108] [Blog Comments]
# Rider - Automatically Nesting JSON Configuration Files

I've been using Visual Studio for a very long time.  Probably close to two decades.  It has it's strengths and weaknesses, but I've always been able to get the job done.  I've been using Rider for a few months now and I'm really enjoying it.  I made the switch partially to try something new, and partially from some mild peer pressure from all the young guns in the office.  It's a great IDE and I'm finding it to be a great alternative to Visual Studio.  Recently, I noticed that JSON configuration files (as are commonly found in ASP.NET Core projects) were not nested like they normally are in Visual Studio.  I googled around for a solution, but after a reading a few inadequate solutions, I decided to write my own.  I'm sharing it here in case anyone else is looking for a solution.

https://www.dandoescode.com/blog/rider-automatically-nesting-json-configuration-files

**Comment by tkapa** [comment]
Cheers for the information! I was just starting to get annoyed by them not nesting too!
